docs/notebooks/model-levels.ipynb

```diff
@@ -0,0 +1,2052 @@
+{
```

Comment:
<p>I don't think you need this chart, but it doesn't hurt</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/contrailcirrus/pycontrails/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
4377b6e67178079f6262753d7226d695e0829883